ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -184,7 +184,7 @@ export const CohortSearch = withCurrentCohortSearchContext()(class extends React
   closeSearch() {
     currentCohortSearchContextStore.next(undefined);
     // Delay hiding attributes page until sidebar is closed
-    setTimeout(() => attributesSelectionStore.next(undefined), 500);
+    window.setTimeout(() => attributesSelectionStore.next(undefined), 500);
```

Comment:
Why do some now use `window` and some use `global` (even within the same file, like here)? Can they all just use the same one?

Reply:
Depends on whether `setTimeout()` value was assigned to typed variable and if the file is a non-test UI component `.tsx` file.
At line # 251 in this file, `this.growlTimer` is type of `NodeJS.Timer`, so it should be `global`.

Reply:
We own that variable though, we could just change the type to `number` instead. Alternatively, why can't they all use `global`?

Reply:
Not sure what you mean by saving the work for another PR. My preference is to have consistency, i.e. either all `global` or all `window`, or else explain why it makes sense to have both coexisting in the codebase. Agree - `window.setTimeout` is probably better, but if  changing the types causes too many complications, I'd settle for all `global.setTimeout` instead, and with a TODO or ticket to switch them all to `window.setTimeout`. Having an arbitrary mix seems very confusing to me.

Reply:
Updated PR to use `global.setTimeout` everywhere.

My suggestion was update code to use `global` or `window` but not both in another new PR. 

Quick question, to use `window.setTimeout`, what would be a replacement function instead of node global `clearTimeout` function? like [this](https://github.com/all-of-us/workbench/blob/a1e8e99683864b98318ebbb119edb1d385bbcef4/ui/src/app/pages/analysis/notebook-redirect.tsx#L324). I'll put the suggestion in TODO ticket to change code to use only `window`.